addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -395,16 +409,28 @@
         t-att-data-next-slide-url="'/slides/slide/%s' % (slug(next_slide)) if next_slide else None"
         t-att-data-channel-id="slide.channel_id.id"
         t-att-data-channel-enroll="slide.channel_id.enroll"
+        t-att-data-channel-can-upload="slide.channel_id.can_upload"
         t-att-data-signup-allowed="signup_allowed">
         <div t-foreach="slide_questions" t-as="question"
-            t-att-class="'o_wslides_js_lesson_quiz_question my-5 %s' % ('completed-disabled' if slide_completed else ('disabled' if not slide.channel_id.is_member else ''))" t-att-data-question-id="question['id']" t-att-data-title="question['question']">
-            <div class="h4">
-                <small class="text-muted"><span t-esc="question_index+1"/>. </small> <span t-esc="question['question']"/>
+             t-att-class="'o_wslides_js_lesson_quiz_question mt-3 %s' % ('completed-disabled' if slide_completed else ('disabled' if not slide.channel_id.is_member else ''))"
+             t-att-data-question-id="question['id']" t-att-data-title="question['question']" >
+            <div class="row d-flex mb-2 mx-0">
+                <div class="h4">
+                    <small class="text-muted">
+                        <i class="o_wslides_js_quiz_sequence_handler fa fa-bars mr-1" t-if="slide.channel_id.can_upload" />
+                        <span class="o_wslides_quiz_question_sequence" t-esc="question_index+1"/>.
+                    </small>
+                    <span t-esc="question['question']"/>
+                </div>
+                <div class="ml-auto" t-if="slide.channel_id.can_upload" >
+                    <i class="o_wslides_js_quiz_edit_question fa fa-pencil-square-o p-1"></i>
+                    <i class="o_wslides_js_quiz_delete_question fa fa-trash p-1"></i>
+                </div>
             </div>
             <div class="list-group">
                 <t t-foreach="question['answers']" t-as="answer">
                     <a t-att-data-answer-id="answer['id']" href="#"
-                        t-att-data-text="answer['text_value']"
+                        t-att-data-text="answer['text_value']" t-att-data-is-correct="answer['is_correct'] if slide.channel_id.can_upload else None"
```

Comment:
Maybe the `is_correct` value can be put to None by the controller instead?
Because it should not be available in the template at all if the user is not supposed to see it.